.drone.yml

```diff
@@ -0,0 +1,35 @@
+---
+kind: pipeline
+type: docker
+name: default
+
+steps:
+
+  - name: k3d_lint
+    image: golang:1.14
+    commands:
```

Comment:
probably want a `when` section in each of these steps so it's not running every time a gh webhook fires e.g. https://github.com/rancher/dapper/blob/master/.drone.yml#L19-L23

Reply:
:heavy_check_mark: